src/main/resources/errors.yaml

```diff
@@ -34,6 +34,10 @@
 # Each error object has:
 # - scope: UPPER_SNAKE_CASE_1
 # - code: UPPER_SNAKE_CASE_1
+# - http_response_override: (optional) The HTTP status code to return when this error is thrown. If not present, the
```

Comment:
Any reason why we want to use "snake_case" instead of camelCase? In Java land, snake_case is rarely used except for interoperability with systems written in Python etc?
But here we control the definition.

Reply:
discussed offline, I thought that was the standard but we have agreed it should be `kebab-case` will create a seperate PR to change